gusto/__init__.py

```diff
@@ -3,6 +3,7 @@
 from gusto.advection import *       # noqa
 from gusto.diagnostics import *     # noqa
 from gusto.diffusion import *     # noqa
+from gusto.transport_equation import *     # noqa
```

Comment:
advection.py and transport_equation.py both define `Advection` classes that are imported into the global namespace here.  That's a bit odd... (the latter overrides the former).